### DISCUSSION TITLE
Browser Spellcheck not triggering ?

### DISCUSSION THREAD

**Discussion by zenhwan** [SafeExamBrowser/seb-win-refactoring#944] [General]
hello our developers have made an update on ckeditor for spellcheck while also creating a custom context menu that removes any word replacement/recommendations and only shows copy/paste as context menu options. 

Currently we are trying cross-browsers and noted none of the created functionality is working on SEB 2.4 or SEB 3.7 on Win/3.3 on Mac. 

The spellcheck used on ck editor relies on the browser spell check. Even with the spellcheck option enabled in the config file, the spell check and context menu do not work as expected. Is there any assistance you can provide?

**Comment by dbuechel** [comment]
First, we strongly recommend to update to the latest, official release version of SEB 3.x and to not use SEB 2.x anymore. Regarding your issue: You can enable the developer tools for the internal browser engine which should help you in debugging, see https://safeexambrowser.org/windows/win_usermanual_en.html#AppearancePane > Enable browser window toolbar > Show developer tools.

**Reply by zenhwan** [reply]
Our Mac dev has informed me with the 3.1 and 3.3 versions of SEB, the spellcheck/recommended text config file options seem to be for Windows only? And our testing shows that the red underline seems to be disabled while the spelling recommendations are available if enabled on the config file settings. 

My question are: 
1. Is the Mac SEB spellcheck highlight functional at all or disabled?
2. Are we able to enable the red underline while hiding the spelling recommendations through CKeditor

for instance this html file with a text field doesn't get picked up by the spellcheck on SEB for mac. 

![image](https://github.com/user-attachments/assets/6c476879-1b15-436c-be02-d48d29a13e92)

**Reply by dbuechel** [reply]
For macOS- and iOS-related questions, you'd need to place them over in the respective repository: https://github.com/SafeExamBrowser/seb-mac.
